components/Auction/Auction.module.css

```diff
@@ -1,153 +1,167 @@
-  font-size: 1rem;
-  width: 150px;
+    border: 0.5px solid #ccc;
+    border-radius: 5px;
+    padding: 10px;
+    margin-bottom: 15px;
+    font-size: 1rem;
+    width: 150px;
 }
 
 .bidBtn {
-  height: 50px;
-  width: 150px;
-  margin-top: 10px;
-  color: white;
-  border-color: rgba(27, 31, 35, 0.15);
-  border-radius: 6px;
-  background-color: #42a742;
+    height: 50px;
+    width: 150px;
+    margin-top: 10px;
+    color: white;
+    border-color: rgba(27, 31, 35, 0.15);
+    border-radius: 6px;
+    background-color: #42a742;
 }
 
 .bidBtn:hover {
-  height: 50px;
-  box-shadow: 0 0 15px -7px rgba(0,0,0,.65);
-  cursor: pointer;
+    height: 50px;
+    box-shadow: 0 0 15px -7px rgba(0, 0, 0, .65);
+    cursor: pointer;
 }
 
 .bidders {
-  display: flex;
-  flex-wrap: wrap;
-  width: 25%;
+    display: flex;
+    flex-wrap: wrap;
+    width: 25%;
 }
 
 .profilePhoto {
-  border-radius: 50%;
-  width: 100%;
+    border-radius: 50%;
+    width: 100%;
 }
 
 .biddersImg {
-  margin-left: 10px;
+    margin-left: 10px;
 }
 
-.biddersImg[data-columns="2"]{
-  width: calc(160px/2);
+.biddersImg[data-columns="2"] {
+    width: calc(160px/2);
 }
-.biddersImg[data-columns="3"]{
-  width: calc(160px/3);
+
+.biddersImg[data-columns="3"] {
+    width: calc(160px/3);
 }
-.biddersImg[data-columns="4"]{
-  width: calc(160px/4);
+
+.biddersImg[data-columns="4"] {
+    width: calc(160px/4);
 }
-.biddersImg[data-columns="5"]{
-  width: calc(160px/5);
+
+.biddersImg[data-columns="5"] {
+    width: calc(160px/5);
 }
-.biddersImg[data-columns="6"]{
-  width: calc(160px/6);
+
+.biddersImg[data-columns="6"] {
+    width: calc(160px/6);
 }
-.biddersImg[data-columns="7"]{
-  width: calc(160px/7);
+
+.biddersImg[data-columns="7"] {
+    width: calc(160px/7);
 }
 
 @media only screen and (max-width: 1200px) {
-  .auctionContainer {
-    width: 85%;
-  }
-
-  .bidBtn {
-    width: 100px;
-  }
+    .auctionContainer {
+        width: 85%;
+    }
+    .bidBtn {
+        width: 100px;
+    }
 }
 
-@media only screen and (max-width: 800px) and (min-width: 900px) { 
-  .bidders {
-    width: 30%;
-  }
+@media only screen and (max-width: 800px) and (min-width: 900px) {
+    .bidders {
+        width: 30%;
+    }
 }
 
 @media only screen and (max-width: 800px) {
-  .auctionContainer {
-    flex-direction: column;
-    width: 60%;
-  }
-
-  .itemInfo {
-    justify-content: center;
-  }
-
-  .bidBtn {
-    width: 150px;
-    margin-bottom: 20px;
-  }
-
-  .bidders {
-    width: 80%;
-    justify-content: center;
-  }
+    .auctionContainer {
+        flex-direction: column;
+        width: 60%;
+    }
+    .itemInfo {
+        justify-content: center;
+    }
+    .bidBtn {
+        width: 150px;
+        margin-bottom: 20px;
+    }
+    .bidders {
+        width: 80%;
+        justify-content: center;
+    }
 }
 
 @media only screen and (max-width: 500px) {
-  .auctionContainer {
-    width: 90%;
-  }
+    .auctionContainer {
+        width: 90%;
+    }
```

Comment:
A couple of things on this file

- I see a mixture of `rem` and `px`. can we please stick to `rem`?
- Also please add an empty line after the end of this file
- Can you also make sure the tab width is set to 2 and not 4? Currently I can see with the code format, the tab spaces has taken 4 into consideration

Reply:
Converted to 'rem'.
When I'm adding the new empty line at the end of the file, it's automatically coming back to the previous line number 167 and declining the new line.
Tab width is already set to 2. Actually in line 165, tab width is showing 4 maybe because it is considering tab width from .auctionContainer.